File: Example/DramAdsObjcExample/DramAdsObjcExample/ViewConrollers/Ads/DramAdsViewController/DramAdsViewController.m
//
//  DramAdsViewController.m
//  DramAdsObjcExample
//
//  Created by Khoren Asatryan on 23.09.23.
//

#import "DramAdsViewController.h"
#import "DramAds/DramAds-Swift.h"

@interface DramAdsViewController ()

@end

@implementation DramAdsViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    DMEnvironment *enviorment = [[DMEnvironment alloc] initWithType: DMEnvironmentTypeDemo];
    [DMDram.sharedManager configureWithEnvironment:enviorment];
}


@end

File: DramAdsIosSdkExample/ViewControllers/DramAdsViewController.swift
//
//  DramAdsViewController.swift
//  DramAdsIosSdkExample
//
//  Created by Khoren Asatryan on 09.09.23.
//

import UIKit
import DramAds

class DramAdsViewController: UITableViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        let enviorment = DM.Enviorment(baseUrl: "https://ads.epomtestsite.com")
        DM.shared.configure(enviorment: enviorment)
    }
    
}
